diff -ur ebtables-v2.0.6.orig/include/linux/netfilter_bridge.h ebtables-v2.0.6/include/linux/netfilter_bridge.h
--- ebtables-v2.0.6.orig/include/linux/netfilter_bridge.h	2003-11-02 19:22:56.000000000 +0100
+++ ebtables-v2.0.6/include/linux/netfilter_bridge.h	2004-09-21 00:28:41.901066448 +0200
@@ -4,7 +4,7 @@
 /* bridge-specific defines for netfilter. 
  */
 
-#include <linux/config.h>
+//#include <linux/config.h>
 #include <linux/netfilter.h>
 #if defined(__KERNEL__) && defined(CONFIG_BRIDGE_NETFILTER)
 #include <asm/atomic.h>
